templates/systemd-override.erb

```diff
@@ -1,6 +1,3 @@
-<%- if @os['name'] == 'Fedora' and @os['release']['major'] <= '31' -%>
-.include /usr/lib/systemd/system/<%= @service_name %>.service
-<% end -%>
```

Comment:
in the future we should convert this to epp

Reply:
we could also do this with systemd::manage_dropin. but I put the conversion to EPP on the list

Reply:
will refactor this in an other branch